### DISCUSSION TITLE
Strapi + nuxt.js without backend deploy

### DISCUSSION THREAD

**Discussion by bhuisman1992** [strapi/strapi#6757] [Legacy Topics]
Hi Guys,

I'm new to Strapi so excuse me if this is a dumb question.
I've worked with Nuxt before (all content was hard coded), but now i want to use strapi as an backend CMS. 

I'm wondering if the following is possible:
1. Run strapi locally
2. Create the front end using Nuxt.js
3. Fill my website using strapi
4. After this is done generate a static version of the site (including all images uploaded through strapi)
5. And after this is done only deploy on netlify. 

So i don't want/need an online backend server.. 
Is this possible with Strapi? Or should I host Strapi's backend on Heroku?

**Comment by jonathanmach** [comment]
Nuxt.js recently released a Full Static generate feature which I believe can be used in your scenario.
